### DISCUSSION TITLE
Is the frontend configured for Admin only?

### DISCUSSION THREAD

**Discussion by sanjiblamichhane** [fastapi/full-stack-fastapi-template#1289] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
NA
```

### Description

The code structure in https://github.com/fastapi/full-stack-fastapi-template/tree/master/frontend feels like the template application is tailored for the admin of the application. But in real world, an application will have customer facing UI and administrator facing UI. To me, this UI feels like it's more tailored towards the admin only. 

Please help me understand what might be a good code structure for a well-integrated admin facing as well as customer facing application developed on top of this template. 

### Operating System

Other

### Operating System Details

NA

### Python Version

NA

### Additional Context

I feel like the frontend is the equivalent of auto-generated admin panel (much like of Django). Please add more suggestion for the problem above.

**Comment by FlaccidFacade** [comment]
This is wild. I don't understand the problem. Maybe look into a privileges model and least privilege practices?

**Reply by sanjiblamichhane** [reply]
This problem is not related to least privilege practices (RBAC).

**Reply by sanjiblamichhane** [reply]
@tiangolo - please advise.

**Comment by alejsdev** [comment, marked as answer]
Hi @sanjiblamichhane :wave: 

Thank you for your feedback. You're right that this template (https://fastapi.tiangolo.com/project-generation/) is initially structured with a focus on administrative functionalities. However, it's designed to be flexible and customizable, serving as a starting point for various types of applications. The idea is that users can adapt it to their specific needs, whether for an internal interface, or a more complex application. You can easily extend it by adding new features depending on your project requirements. I hope this has clarified things for you.

**Reply by sanjiblamichhane** [reply]
Hi @alejsdev - Thanks for your response and clarification.  I am quite impressed with this template already and the design thoughts put into it. Therefore, I would like to follow up on this from above question.
> Please help me understand what might be a good code structure for a well-integrated admin facing as well as customer facing application developed on top of this template. 

A general guideline or a template page would be a great starting point. Thanks.
